src/script/script_error.h

```diff
@@ -59,12 +59,15 @@ typedef enum ScriptError_t
     SCRIPT_ERR_MINIMALDATA,
     SCRIPT_ERR_SIG_PUSHONLY,
     SCRIPT_ERR_SIG_HIGH_S,
-    SCRIPT_ERR_SIG_NULLDUMMY,
+    SCRIPT_ERR_MULTISIG_HINT,
     SCRIPT_ERR_PUBKEYTYPE,
 
     /* softfork safeness */
     SCRIPT_ERR_DISCOURAGE_UPGRADABLE_NOPS,
 
+    /* require valid signaturesel */
```

Comment:
"signaturesel"?

Reply:
Ugh, my MacBook Pro is suffering that famous key-repeat issue, so there are soo many typos in things I've been writing recently. I can't wait to get it fixed. Thanks for catching another one.

Reply:
Ahh, that explains it. More of these in the other review.